metadata/demographic/census_blocks_2020.md

```diff
@@ -0,0 +1,90 @@
+<!-- The filename should match the OpenSGID layer_name. For example, roads.md. The category will be inferred from the directory structure -->
+
+# Title
+
+<!-- The human-readable layer name that should match the pageTitle item in the data page's metadata. If it's a statewide layer, it should include "Utah", e.g. "Utah Roads" -->
+Utah Census 2020 Blocks
+
+## Brief Summary
+
+<!-- A super-short, one-liner description of the dataset. Corresponds to the pageDescription item in the data page's metadata -->
+Basic demographic data in the smallest geographic unit used by the US Census Bureau.
+
+## Summary
+
+<!-- A brief (<2048 characters for AGOL) explanation of the dataset to give the user a high-level overview of what the layer is when skimming lists of datasets. Will be the first description people see in Hub open data and should match the Summary section of the layer's data page  -->
+Census Blocks are the smallest geographic unit used by the United States Census Bureau for tabulating population information. Due to privacy concerns, only a limited amount of data is available at the block level.
+
+## Description
+
+<!-- A more in-depth explanation of the dataset, where it came from and how its created, so the user can decide if its what they need. -->
+
+### What is the dataset
+
+Census blocks are small geographic zones through which population data can be collected and organized. These blocks are often bounded by physical features such as roads or rivers, as well as municipal boundaries such as city limits or property lines. This dataset includes the census blocks for the State of Utah, current as of the 2020 Census and represented as polygons.
```

Comment:
I can mention it in the updates, however, I believe that when the next census is released, we will be making a new layer with its own metadata for it (rather than adding the 2030 census blocks to this one). Thus, although the layer may be updated, the fact that these are the 2020 census blocks is unlikely to change, which is why I included it as part of "what is the dataset" rather than "update schedule". Does that make sense?